setup.py

```diff
@@ -0,0 +1,24 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+
+setuptools.setup(
+    name="commands-base",
+    version="2.0.1",
+    author="Fivestars",
+    author_email="dev@fivestars.com",
+    description="Base command for aladdin commands",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/fivestars-os/commands-base",
+    packages=[
+        "commands_base"
+    ],
+    entry_points={
+        "console_scripts": [
+            "aladdin_command = commands_base:main",
+        ]
+    },
+)
```

Comment:
I had added this comment but then GitHub removed it because the file where I made it changed.

This file is needed because `poetry install` installs our package (commands-base) in `dev` mode, the equivalent of doing `pip install -e`, which means a symlink is created that points to the original source code location, that link is then invalid because we are using docker multi-stage and the code is not there any more (I don't think we should rely on the symlink anyways). Poetry is not meant to be used as a package deployment tool, hence the use of `pip install` instead to install `commands-base` into `site-packages`

More info here:
https://github.com/python-poetry/poetry/issues/1333

Reply:
Right, but what I'm saying is, can't you pip install from a pyproject.toml?  As linked above, it looks like pip may support this file now.

Reply:
Just confirmed by pip installing the 'common' package from messaging-edge, which has no setup.py.

Reply:
Looks like it's not quite as simple as that, and I'll have to read more about how that works.  Yes, pip can parse and load information from that file, but the information we currently store in it is poetry-specific.  There may be an answer where both tools read from the same file, but in the meantime you can definitely do

```
poetry build
pip install .
```

poetry build creates the wheel in the dist/ directory, then pip install . installs that directly (ie, it doesn't try to build it itself).  That way, the package metadata lives in one place, and pip is just responsible for installing the built package.

*That* said, my question about why doing poetry install isn't sufficient for our needs here still stands :)